README.md

```diff
@@ -2,6 +2,7 @@
 [![Build Status](https://travis-ci.org/Statoil/camille.svg?branch=master)](https://travis-ci.org/Statoil/camille)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/61db6e0137e743db84041b1239436c39)](https://www.codacy.com/app/equinor_sib/camille?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Statoil/camille&amp;utm_campaign=Badge_Grade)
 [![Codacy Badge](https://api.codacy.com/project/badge/Coverage/61db6e0137e743db84041b1239436c39)](https://www.codacy.com/app/equinor_sib/camille?utm_source=github.com&utm_medium=referral&utm_content=Statoil/camille&utm_campaign=Badge_Coverage)
+[![MutMut Badge](https://img.shields.io/badge/dynamic/json.svg?label=Mutant%20survival%20rate&url=https%3A%2F%2Fs3-eu-west-1.amazonaws.com%2Fequinor-sib%2Fcamille%2Fmutmut_report.json&query=%24..survival_rate&colorB=blue&suffix=%20%25)](https://mutmut.readthedocs.io/en/latest/)
```

Comment:
Just to be nasty: 
1. all other badge names start lowercase :smile:
2. link in all other badges leads to actual project results, so maybe ours should point out to build too? ~~Unless this is confidential~~

Reply:
1. Well spotted, correcting.
2. This is running on Jenkins which isn't reachable for others. The alternative would be to point to our builds on Travis, but we don't have anything relating to mutation testing. So I think it's an OK alternative to use opportunity to point to a mutation test tool that is proven to work on this project.